README.md

```diff
@@ -12,13 +12,38 @@ npm run serve
 
 ## Usage
 
-* Click "Upload Edge List CSV" to upload a file of the form:
+* Click "UPLOAD CVS OR JSON" to upload a file of the form:
 ```
 source,target
 a,b
 b,c
 c,d
 d,a
 ```
+for CSV and
+```
+{
+  "edges": [
+    {
+      "source": "a",
+      "target": "b"
+      },
+    {
+      "source": "b",
+      "target": "c"
+    },
+    {
+      "source": "c",
+      "target": "d"
+    },
+    {
+      "source": "d",
+      "target": "a"
+    }
+  ]
```

Comment:
This should also include a node list, e.g.:

```
"nodes": [
  { "id": "a" },
  { "id": "b" },
  { "id": "c" },
  { "id": "d" }
],
```

Reply:
Fixed with c9aaea399ba4246d63a0c27a18c042a8b2da84c6